Core/TimberApi/AssetShaderSystem/AssetShaderFixer.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using UnityEngine;
+
+namespace TimberApi.AssetShaderSystem
+{
+    internal class AssetShaderFixer
```

Comment:
Shouldn't Tobbert's shader fix be here?

Reply:
Ah, seems like it's implemented somewhere else

Reply:
Yes the shader fixes (except the default) are now dynamicly addable within it's own classes :)